shared/rakefiles/xk_infra.rake

```diff
@@ -98,7 +98,8 @@ task :apply_common_infra => [@gcp_creds_file] do
    "cloudbilling.googleapis.com",
    "iam.googleapis.com",
    "dns.googleapis.com",
-   "compute.googleapis.com"].each do |service|
+   "compute.googleapis.com",
+   "securitycenter.googleapis.com"].each do |service|
```

Comment:
is the `securitycenter.googleapis.com` needed a the creation time of the organization or is it needed to be at org level?. Otherwise I'll add it to the gcp-project TF module.

Reply:
Command center has global scope, so it needs to be enabled in one project per organization.